### ISSUE TITLE
Constant red light when following Getting Started guide (sd card option)

### ISSUE THREAD

**Issue by mentarus** [google-coral/edgetpu#294]
Hi, I've been trying to get my Coral EdgeTPU board working as per the [getting started guide.](https://coral.ai/docs/dev-board/get-started).
I've verified the SHA256 of the download (enterprise-eagle-flashcard-20201208133741.zip) and it matches, balenaEtcher also verifies the with no errors (multiple times) I've tried this a few times to no avail (the red light stays on). The serial output is below. Would appreciate pointers as to what could be wrong.

```
U-Boot SPL 2019.04.1 (Apr 29 2020 - 18:40:05 +0000)
power_bd71837_init
Board id: 6
DDRINFO: start DRAM init
DDRINFO:ddrphy calibration done
DDRINFO: ddrmix config done
Normal Boot
Trying to boot from MMC2
hdr read sector 300, count=1


U-Boot 2019.04.1 (Apr 29 2020 - 18:40:05 +0000), Build: jenkins-enterprise.uboot-imx-1

CPU:   Freescale i.MX8MQ rev2.0 1500 MHz (running at 1000 MHz)
CPU:   Commercial temperature grade (0C to 95C) at 28C
Reset cause: POR
Model: Freescale i.MX8MQ Phanbell
DRAM:  1 GiB
MMC:   FSL_SDHC: 0, FSL_SDHC: 1
Loading Environment from MMC... *** Warning - bad CRC, using default environment

In:    serial
Out:   serial
Err:   serial

 BuildInfo:
  - ATF
  - U-Boot 2019.04.1

flash target is MMC:0
Net:
Warning: ethernet@30be0000 using MAC address from ROM
eth0: ethernet@30be0000
Fastboot: Normal
Normal Boot
Hit any key to stop autoboot:  0
## Executing script at 40480000
Wrong image format for "source" command
## Starting auxiliary core at 0x00000000 ...

```

**Comment by Namburger** [comment]
@mentarus sorry about that, it hard to tell exactly what went wrong, could you try manual flashing from our flash script as described here instead?
https://coral.ai/docs/dev-board/reflash/#flash-the-board

**Comment by mentarus** [comment]
Thanks for the quick response @Namburger, tried that earlier on windows but never got the fastboot to see the device (yes I did go into fastboot mode as I have the old version of the board) tries on two windows machines with the same effect. Eventually tried on Linux and it fastboot saw the device straight away. Would be good to understand why the default method didn't work though!

**Comment by Namburger** [comment]
@mentarus I see, I have a feeling that the sdcard slot might be faulty, my suggestion is to check and see if you can mount anything form that sdcard slot at all. If it is ends up being a defective HW, we can definitely replace

**Comment by j2abro** [comment]
I too followed the getting started guide and got the identical output as @mentarus. I tried this on 2 different Coral boards and got the same output as shown above. I have successfully installed a number of boards using the previous (non-SDcard) method.

**Comment by j2abro** [comment]
I solved it. I tried burning the flash image onto another SD card (new, out of box) and the install worked. So it looks like the problem was with the SD card. The previous failed attempt was using a card that had already been used so perhaps that was the issue. I posted the question and answer on SO as well:
[https://stackoverflow.com/questions/65745058/flashing-google-coral-board-from-sd-card-fails-with-error-wrong-image-format-fo/65745059#65745059](url)
